### PR TITLE
Rename `suppress_whitespace` into `whitespace`

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
As discussed in https://github.com/djc/askama/issues/671.

That'll make what follows simpler.

### CODE REVIEW

File: askama_shared/src/lib.rs [hunk 1]
```diff
@@ -302,12 +306,33 @@ impl RawConfig<'_> {
...
 }
 
+#[derive(Clone, Copy, PartialEq, Eq, Debug)]
+#[cfg_attr(feature = "serde", derive(Deserialize))]
+#[cfg_attr(feature = "serde", serde(field_identifier, rename_all = "lowercase"))]
+pub(crate) enum WhitespaceHandling {
+    /// The default behaviour. It will leave the whitespace characters "as is".
+    Preserve,
+    /// It'll remove all the whitespace characters before and after the jinja block.
+    Trim,
```
Comment (djc) [inline askama_shared/src/lib.rs:316]: `Suppress`, right?

File: askama_shared/src/lib.rs [hunk 2]
```diff
@@ -118,7 +118,7 @@ struct Config<'a> {
     syntaxes: BTreeMap<String, Syntax<'a>>,
     default_syntax: &'a str,
     escapers: Vec<(HashSet<String>, String)>,
-    suppress_whitespace: bool,
+    whitespace_handling: WhitespaceHandling,
```
Comment (djc) [inline askama_shared/src/lib.rs:121]: Let's just call this `whitespace`, too.

File: askama_shared/src/lib.rs (hunk 1 above)
Comment (GuillaumeGomez) [reply askama_shared/src/lib.rs:316]: Oh yes indeed! Sorry about that.

**Comment by GuillaumeGomez** [comment]
Updated!

**Review by djc — APPROVED** [review]

**Pull request merged**